dev_guide/application_ephemeral_storage_sizing.adoc

```diff
@@ -0,0 +1,143 @@
+[[dev-guide-application-ephemeral-storage-sizing]]
+= Application ephemeral storage sizing
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+_NOTE: This only applies if your administrator has enabled the ephemeral-storage tech preview in {product-title} 3.10._
+
+This page is intended to provide guidance to application developers using
+{product-title} on:
+
+. Determining the ephemeral storage and risk requirements of a containerized application
+   component and configuring the container ephemeral storage parameters to suit those
+   requirements.
+
+. Configuring containerized application runtimes (for example, OpenJDK) to adhere
+   optimally to the configured container ephemeral storage parameters.
+
+. Diagnosing and resolving ephemeral storage-related error conditions associated with
+   running in a container.
+
+== Background
+
+It is recommended to read fully the overview of how {product-title} manages
+xref:compute_resources.adoc#dev-compute-resources[Compute Resources] before
+proceeding.
+
+For the purposes of sizing application ephemeral storage, the key points are:
+
+* For each kind of resource (memory, cpu, storage, and ephemeral storage), {product-title} allows
+  optional *request* and *limit* values to be placed on each container in a
+  pod. For the purposes of this page, we are solely interested in ephemeral storage
+  requests and limits.
+
+* *Ephemeral Storage request*
+
+  - The ephemeral storage request value, if specified, influences the {product-title}
+    scheduler. The scheduler considers the ephemeral storage request when scheduling a
+    container to a node, then fences off the requested ephemeral storage on the chosen node
+    for the use of the container.
+
+* *Ephemeral Storage limit*
+
+  - The ephemeral storage limit value, if specified, provides a hard limit on the ephemeral storage
+    that can be allocated across all the processes in a container.
+
+  - If both ephemeral storage request and limit are specified, the ephemeral storage limit value must
+    be greater than or equal to the ephemeral storage request.
+
+* *Administration*
+
+  - The cluster administrator may assign quota against the ephemeral storage request value,
+    limit value, both, or neither.
+
+  - The cluster administrator may assign default values for the ephemeral storage request
+    value, limit value, both, or neither.
+
+  - The cluster administrator may override the ephemeral storage request values that a
+    developer specifies, in order to manage cluster overcommit. This occurs on
+    OpenShift Online, for example.
+
+== Strategy
```

Comment:
This heading needs to be more specific. Would "Sizing application ephemeral storage" or "How to size application ephemeral storage" be correct?

Reply:
The wording is parallel to that used in `dev_guide/application_memory_sizing.adoc`.

Reply:
Resolved.